### ISSUE TITLE
Не сохраняются адреса прокси

### ISSUE THREAD

**Issue by Pochemuzhe** [ramjke/Translumo#139]
Не сохраняются адреса прокси 
![image](https://github.com/user-attachments/assets/b40e3def-5ea5-435d-b902-7a346dfe3ebc)
Нажимаешь применить, они слетают вс равно.

**Comment by evil-pechenka** [comment]
Да, та же проблема, вводил несколько раз — ничего не сохраняется.

Поле ввода очень странное, с подчёркиванием и глючит.

**Comment by x4tz1s** [comment]
I had the same problem; I was using proxy servers that I found online without a password. 

I just bought one for 2 euros a month. 

It had all the details along with passwords, I entered them, and it works perfectly.

**Comment by ramjke** [comment]
Duplication of #143
